sp800-63b/sec6_lifecycle.md

```diff
@@ -4,75 +4,75 @@
 
 *This section is normative.*
 
-During the lifecycle of an authenticator bound to a subscriber's identity, a number of events can occur that affect the use of that authenticator. These events include binding, loss, theft, unauthorized duplication, expiration, and revocation. This section describes the actions to be taken in response to those events.
+During the lifecycle of a subscriber's authenticator, a number of events can occur that affect the use of that authenticator. These events include binding, loss, theft, unauthorized duplication, expiration, and revocation. This section describes the actions to be taken in response to those events.
 
 ### <a name="binding"></a>6.1 Authenticator Binding
 
-*Authenticator binding* refers to the establishment of an association between a specific authenticator and a subscriber's account, enabling the authenticator to be used, possibly in conjunction with other authenticators, to authenticate for that account.
+*Authenticator binding* refers to the establishment of an association between a specific authenticator and a subscriber's account, enabling the authenticator to be used - possibly in conjunction with other authenticators - to authenticate for that account.
 
 Authenticators SHALL be bound to subscriber accounts by either:
 
-- issuance by the CSP as part of enrollment, binding, or other processes, or
-- a subscriber-provided authenticator that is acceptable to the CSP.
+- issuance by the CSP as part of enrollment.
```

Comment:
Understood, but saying that one SHALL bind, by binding is circular. Ok...my wife is outside waiting for me...got to run.